.circleci/config.yml

```diff
@@ -363,36 +402,54 @@ commands:
                   --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
                   --num-uniform-shards 3
               else
-                gcloud beta firebase test android run --type instrumentation \
-                --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
-                --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
-                --device model=harpia,version=23,locale=en,orientation=portrait \
-                --device model=blueline,version=28,locale=en,orientation=portrait \
-                --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
-                --num-uniform-shards 4
+                if << parameters.nightly >> ; then
+                  gcloud beta firebase test android run --type instrumentation \
+                    --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
+                    --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
+                    --device model=harpia,version=23,locale=en,orientation=portrait \
+                    --device model=blueline,version=28,locale=en,orientation=portrait \
+                    --device model=redfin,version=30,locale=en,orientation=portrait \
+                    --device model=oriole,version=33,locale=en,orientation=portrait \
+                    --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
+                    --num-uniform-shards 4
+                else
+                  gcloud beta firebase test android run --type instrumentation \
+                    --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
+                    --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
+                    --device model=redfin,version=30,locale=en,orientation=portrait \
+                    --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
+                    --num-uniform-shards 4
+                fi
               fi
             fi
 
   run-firebase-robo:
+    parameters:
+      nightly:
```

Comment:
Nit: maybe a better name would be `multiple-devices` (or `single-device`). It doesn't really matter if it's nightly or not.